bundles/org.dataflowanalysis.analysis.pcm.informationflow/src/org/dataflowanalysis/analysis/pcm/informationflow/core/seff/IFStartSEFFPCMVertex.java

```diff
@@ -0,0 +1,139 @@
+				extractionStrategy);
+	}
+
+	@Override
+	protected AbstractIFSEFFPCMVertex<StartAction> createIFSEFFVertex(StartAction element,
+			List<? extends AbstractPCMVertex<?>> previousElements, Deque<AssemblyContext> context,
+			List<Parameter> parameter, ResourceProvider resourceProvider, boolean considerImplicitFlow,
+			IFPCMExtractionStrategy extractionStrategy) {
+		return new IFStartSEFFPCMVertex(element, previousElements, context, parameter, resourceProvider,
+				considerImplicitFlow, extractionStrategy);
+	}
+
+	@Override
+	protected List<DataFlowVariable> modifyIncomingDataFlowVariables(List<DataFlowVariable> incomingDataFlowVariables) {
+		if (!isConsideringImplicitFlow()) {
+			return filterCallParameters(incomingDataFlowVariables);
+		}
+		// incoming variables are the outgoing variables of the branch if existent
+		List<DataFlowVariable> incomingVariables = incomingDataFlowVariables;
+		if (isElementInGuardedBranchTransitionSEFF()) {
+			incomingVariables = evaluateOutgoingDataFlowVariablesOfBranchTransition(incomingDataFlowVariables);
+		}
+
+		// Avoid parameter filter for security context layers in StartVertex
+		List<DataFlowVariable> securityContextLayers = IFSecurityContextUtils
+				.getAllSecurityContextLayers(incomingVariables);
+		List<DataFlowVariable> modifiedIncoming = filterCallParameters(incomingVariables);
+
+		modifiedIncoming = new ArrayList<>(modifiedIncoming);
+		for (DataFlowVariable securityContextLayer : securityContextLayers) {
+			if (!modifiedIncoming.contains(securityContextLayer)) {
+				modifiedIncoming.add(securityContextLayer);
+			}
+		}
+		return modifiedIncoming;
+	}
+
+	@Override
+	protected List<VariableCharacterisation> extractVariableCharacterisations() {
+		return new ArrayList<VariableCharacterisation>();
+	}
+
+	@Override
+	protected void checkConfidentialityVariableCharacterisations(
+			List<ConfidentialityVariableCharacterisation> characterisations) {
+		return;
+	}
+
+	@Override
+	protected List<DataFlowVariable> modifyOutgoingDataFlowVariables(List<DataFlowVariable> outgoingVariables) {
+		return outgoingVariables;
+	}
+
+	/*
+	 * Evaluation of GuardedBranchTransition
+	 */
+
+	private List<DataFlowVariable> evaluateOutgoingDataFlowVariablesOfBranchTransition(
+			List<DataFlowVariable> incomingVariables) {
+		var supposedGuardedBranchTransition = getReferencedElement().eContainer().eContainer();
+		if (supposedGuardedBranchTransition instanceof GuardedBranchTransition bT) {
+
+			List<ConfidentialityVariableCharacterisation> confChars = extractCvcsOfGuardedBranchTransition(bT);
+			List<DataFlowVariable> outgoingVariables = getDataFlowVariables(getVertexCharacteristics(), confChars,
+					incomingVariables);
+			return outgoingVariables;
+		}
+		return incomingVariables;
+	}
+
+	private List<ConfidentialityVariableCharacterisation> extractCvcsOfGuardedBranchTransition(
+			GuardedBranchTransition bT) {
```

Comment:
Nitpick: Please avoid using abbreviations for variable names

Reply:
Changed